src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -87,6 +94,14 @@ public LinkedFileViewModel(LinkedFile linkedFile,
         this.externalFileTypes = externalFileTypes;
         this.xmpPreferences = xmpPreferences;
 
+        fileExistsValidator = new FunctionBasedValidator<>(
+                linkedFile.linkProperty(),
+                link -> {
+                    Optional<Path> path = FileHelper.expandFilename(databaseContext, link, filePreferences);
```

Comment:
What happens if a website url is linked?

Reply:
Yes, it is also possible to link url's. Some fetcher return the link to the PDF which then can be downloaded by the user (right click > download). Also some users add links to their notes in this way (for example onenote url's).